automation/src/test/scala/org/broadinstitute/dsp/consent/requests/Requests.scala

```diff
@@ -94,25 +94,25 @@ object Requests {
       responseContent
     }
 
-    def dataSetCatalog(expectedStatus: Int, userId: String, additionalHeaders: Map[String, String]): HttpRequestBuilder = {
-      http("DataSet Catalog Requests")
+    def datasetCatalog(expectedStatus: Int, additionalHeaders: Map[String, String]): HttpRequestBuilder = {
+      http("Dataset Catalog Requests")
         .get("/")
         .headers(TestConfig.jsonHeader)
         .headers(additionalHeaders)
         .resources(
-          DataSet.byUserId(expectedStatus, userId, additionalHeaders),
+          Dataset.byUserId(expectedStatus, additionalHeaders),
           Dac.list(expectedStatus, additionalHeaders)
         )
     }
   }
 
-  object DataSet {
+  object Dataset {
     val dataSetResponse: String = "DATASET_RESPONSE"
     val dataSetsByDataSetId: String = "DATASET_BYDSID_"
 
-    def byUserId(expectedStatus: Int, userId: String, additionalHeaders: Map[String, String]): HttpRequestBuilder = {
-      http("Get DataSet By User")
-        .get("/api/dataset?userId=" + userId)
+    def byUserId(expectedStatus: Int, additionalHeaders: Map[String, String]): HttpRequestBuilder = {
+      http("Get Dataset By User")
+        .get("/api/dataset")
```

Comment:
This API was changed in https://github.com/DataBiosphere/consent/pull/874 to use the auth user instead of passing in a user id.